extension/flink-doris-connector/src/main/java/org/apache/doris/flink/table/DorisStreamLoad.java

```diff
@@ -133,11 +141,15 @@ public void load(String value) throws StreamLoadException {
     }
 
     private LoadResponse loadBatch(String value) {
-        Calendar calendar = Calendar.getInstance();
-        String label = String.format("flink_connector_%s%02d%02d_%02d%02d%02d_%s",
-                calendar.get(Calendar.YEAR), calendar.get(Calendar.MONTH) + 1, calendar.get(Calendar.DAY_OF_MONTH),
-                calendar.get(Calendar.HOUR_OF_DAY), calendar.get(Calendar.MINUTE), calendar.get(Calendar.SECOND),
-                UUID.randomUUID().toString().replaceAll("-", ""));
+        String label = streamLoadProp.getProperty("label");
+        if(StringUtils.isBlank(label)){
+            Calendar calendar = Calendar.getInstance();
+            label = String.format("flink_connector_%s%02d%02d_%02d%02d%02d_%s",
```

Comment:
SimpleDateFormat is easiler to use and read

Reply:
already modified